docs/howto/windows.txt

```diff
@@ -125,3 +125,11 @@ Common pitfalls
 
     ...\> set http_proxy=http://username:password@proxyserver:proxyport
     ...\> set https_proxy=https://username:password@proxyserver:proxyport
+
+* In general, Django assumes that ``UTF-8`` encoding is used for I/O. This may
+  cause problems if your system is set to use a different encoding. Recent
+  versions of Python allow setting the :envvar:`PYTHONUTF8` environment
+  variable in order to force a ``UTF-8`` encoding. Windows 10 also provides a
+  system wide setting by checking ``Use Unicode UTF-8 for worldwide language
+  support`` in :menuselection:`Language --> Administrative Language Settings
+  --> Change system locale`.
```

Comment:
Forgive me, but I don't use Windows and I would be guessing if I presumed that this was in the Windows "Settings" application(?)... Or is it a setting in PowerShell or CMD? Perhaps this should finish with `...in the Settings application.` Although I get the feeling that "application" might be the wrong word to use here.

Reply:
I'm not sure what I'd call all of the menus / options are anymore. (Bring back the control panel?)

What you're looking for here is the language settings. I typed language into the search bar and you get a big button with the option you're looking for. The UI then calls this box 'language' and not 'language settings' 🤷

I think if you're going down this rabbit hole then this guidance is ok, especially as the previous option is 'just set an env var'. I tried to search for a Microsoft page we could link to but search skills failed to find anything we could use.

Reply:
I've adding `in system settings`.